WordPressAuthenticator/Email Client Picker/LinkMailPresenter.swift

```diff
@@ -26,8 +33,13 @@ class LinkMailPresenter {
     }
 
     private func showAlertToCheckEmail(on viewController: UIViewController) {
-        let title = NSLocalizedString("Please check your email", comment: "Alert title for check your email during logIn/signUp.")
-        let message = NSLocalizedString("Please open your email app and look for an email from WordPress.com.", comment: "Message to ask the user to check their email and look for a WordPress.com email.")
+        let title = NSLocalizedString("Check your email!",
+                                      comment: "Alert title for check your email during logIn/signUp.")
+        let message = NSLocalizedString("We just emailed a link to ",
```

Comment:
I'd suggest using a localized string with format here, so that these strings can be combined into one – different languages may need to put the email address into a different order in the string, so it's easiest for translation if it's all together. Something like:

```
let message = String.localizedStringWithFormat(NSLocalizedString("We just emailed a link to %@. Please check your mail app and tap the link to log in.", comment: "blah"), emailAddress)
```

Reply:
done